### ISSUE TITLE
[BLOCKED BY DST TICKET] FE  Expandable headers double encapsulating html entities in the title

### ISSUE THREAD

**Issue by swirtSJW** [department-of-veterans-affairs/va.gov-cms#11157]
## Statuses
**[2025-01-14]** DST ticket is still in their backlog.
**[2024-04-25]** Carol in DST responded that it's in their backlog but they're working on other priority items since the issue has a workaround (the editor enters the text differently)
**[2024-04-24]** Fran commented on the DST ticket that's blocking this one asking for a status.
**[2024-01-19]** Laura F has confirmed that the linked, blocking ticket is still an issue.

**BLOCKED BY: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1311**

## Description

Title fields in the CMS already encapsulate html (converting & to &amp;)  but it seems like the FE is encapsulating them a second time which leads to displaying things  oddly.
Reported in slack https://dsva.slack.com/archives/CDHBKAL9W/p1665087407151339

![image](https://user-images.githubusercontent.com/5752113/194577983-3d6c3f0a-f11c-4edb-8566-cf178f586d5e.png)

![image](https://user-images.githubusercontent.com/5752113/194578030-9d5d29a6-559f-49dd-a94c-674658eb33c5.png)

## To Reproduce
1. Go to https://www.va.gov/coronavirus-veteran-frequently-asked-questions/#claims-and-applications
2. See the bad text (unless it has been removed in the CMS already

## Tasks
- [ ] Find cause
- [ ] Find original github issue that added the (unnecessary) filter, and discern what problem they were trying to solve
- [ ] fix
- [ ] verify

## AC / Expected behavior
- [ ] a title with a & in it should show like an &
- [ ] a nbsp should appear like a nbsp

 ## Additional context
Not sure if this is a FE filter being applied or perhaps the graphQL using the wrong variant (raw, value, or processed)

## Labels
(You can delete this section once it's complete)
- [x] Issue type (red) (defaults to "Defect")
- [ ] CMS subsystem (green)
- [ ] CMS practice area (blue)
- [x] CMS workstream (orange) (not needed for bug tickets)
- [ ] CMS-supported product (black)

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by apisandipas** [comment]
This issue has me pretty stumped overall. It would appear at first that removing the "encode" filter would fix the issue however that didnt seem to be the case. My understanding is the "encode filter" would take a literal symbol, like the Ampersand and convert it into and HTML Entity like "&amp;", which is really the opposite of what we want. We would normally want to use a decode function to reverse that process, since according to the issue, the CMS already converts the & to  &amp;. That all appears moot in this case as my testing below shows. 

I used the following bit of code below in different areas on the page to gain an understanding of the behavior of these filters:
<img width="541" alt="Screen Shot 2022-10-24 at 10 00 32 AM" src="https://user-images.githubusercontent.com/732460/197560099-19820047-3654-4160-a29f-bb1b145b13e5.png">

This image shows the filters in a regular portion of the page template
<img width="368" alt="Screen Shot 2022-10-24 at 10 00 50 AM" src="https://user-images.githubusercontent.com/732460/197560694-7acdfb9d-541a-4ef5-8cd0-0917b4e9fa11.png">

However in the collable heading, non of those filters function at all:
<img width="740" alt="Screen Shot 2022-10-24 at 10 01 06 AM" src="https://user-images.githubusercontent.com/732460/197560842-6c25c316-bd05-4d43-adbe-c7a68df29dc6.png">

My conclusion is that this has something to do with the transsclusion of HTML in the Design System components, which utilize the Slot element. I couldn't however find anything to out there to back this up, not did anything stand out in the code for the Design Component collapsible component. In short, I think everything we have control over is functioning fine, and it's not something we could fix on our end but would need investigation on the Design System side of things. 

That being said, it appears the offencing HTML entities have been removed from the content on Prod which spurred the issue in the first place. It may suffice for now to add some helpered text on the CMS side asking editors not to include Ampersands or other encodable entities in the Collapsible headers.

**Comment by jilladams** [comment]
Blake proposed filing a defect for the design system: https://dsva.slack.com/archives/C01DBGX4P45/p1666634552606349?thread_ts=1666631691.820379&cid=C01DBGX4P45

**Comment by wesrowe** [comment]
Following that slack link, @jilladams, did you or someone file a defect with the DST?

**Comment by jilladams** [comment]
Indeed: https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1311

Adding as a dependency

**Comment by jilladams** [comment]
@laflannery would you consider this an a11y bug as well? If so, we might be able to use that to nudge design system to fix their ticket.

**Comment by laflannery** [comment]
@jilladams Yes I would consider this an accessibility issue, this is making the Heading/content confusing and possibly for some users non-understandable. Folks, especially those using assistive technology, may not realize that this is actually an error but may think this is some sort of acronym or abbreviations that they are not aware of which will be confusing to them.

**Comment by FranECross** [comment]
Question/comment added to the ticket blocking this one...

**Comment by FranECross** [comment]
Shared this ticket with @chriskim2311 as a candidate for him to look at. The DST ticket has been completed, so it's no longer blocked. [Slack message](https://agilesix.slack.com/archives/C05Q7J61NR5/p1756833564419849?thread_ts=1756816153.059109&cid=C05Q7J61NR5)

**Comment by chriskim2311** [comment]
DST ticket seems to fix this issue. No additional action is needed. Closing!
